### PR TITLE
[se.m3u] - added svt channels.

### PR DESCRIPTION
**Pull request by librazhd7**

### CODE REVIEW

**Review by ghost — DISMISSED** [review]

File: streams/se.m3u [hunk 1]
```diff
@@ -37,3 +37,11 @@ https://csm-e-tv4se-eb.tls1.yospace.com/csm/live/142791553.m3u8
 http://51.91.73.99:25461/sweden/PM66f7Y43H/12476
 #EXTINF:-1 tvg-id="" status="error",OPPNA KANALEN (540p)
 http://83.140.64.214/edge/_451iw2h_/451iw2h/chunklist_w348058882.m3u8
+#EXTINF:-1 tvg-id="SVT1.se" status="online",SVT1 (auto) [Geo-blocked]
```
Comment (LaneSh4d0w) [inline streams/se.m3u:40]:
```suggestion
#EXTINF:-1 tvg-id="SVT1.se" status="online",SVT1 [Geo-blocked]
```

auto not needed

File: streams/se.m3u [hunk 2]
```diff
@@ -37,3 +37,11 @@ https://csm-e-tv4se-eb.tls1.yospace.com/csm/live/142791553.m3u8
...
+https://svt1-c.akamaized.net/se/svt1/master.m3u8
+#EXTINF:-1 tvg-id="SVT2.se" status="online",SVT2 (auto) [Geo-blocked]
```
Comment (LaneSh4d0w) [inline streams/se.m3u:42]:
```suggestion
#EXTINF:-1 tvg-id="SVT2.se" status="online",SVT2 [Geo-blocked]
```
auto not needed

File: streams/se.m3u [hunk 3]
```diff
@@ -37,3 +37,11 @@ https://csm-e-tv4se-eb.tls1.yospace.com/csm/live/142791553.m3u8
...
+https://svt2-c.akamaized.net/se/svt2/master.m3u8
+#EXTINF:-1 tvg-id="SVT24.se" status="online",SVT24 (auto) [Geo-blocked]
```
Comment (LaneSh4d0w) [inline streams/se.m3u:44]:
```suggestion
#EXTINF:-1 tvg-id="SVT24.se" status="online",SVT24 [Geo-blocked]
```

auto not needed

File: streams/se.m3u [hunk 4]
```diff
@@ -37,3 +37,11 @@ https://csm-e-tv4se-eb.tls1.yospace.com/csm/live/142791553.m3u8
...
+https://svtb-c.akamaized.net/se/svtb/master.m3u8
+#EXTINF:-1 tvg-id="" status="online",SVT Kunskapskanalen (auto) [Geo-blocked]
```
Comment (LaneSh4d0w) [inline streams/se.m3u:46]:
```suggestion
#EXTINF:-1 tvg-id="" status="online",SVT Kunskapskanalen [Geo-blocked]
```
auto not needed

**Review by ghost — APPROVED** [review]

**Review by its0x08 — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**